corehq/apps/app_manager/templates/app_manager/v1/summary.html

```diff
@@ -49,6 +49,9 @@
                 caseDownloadURL: '{% url 'download_case_summary' domain app_id %}',
                 formDownloadURL: '{% url 'download_form_summary' domain app_id %}',
                 appDownloadURL: '{% url 'download_app_summary' domain app_id %}',
+                appSettingsURL: '{% url "view_app" domain app.id %}',
+                appHomeURL: '{% url "view_app" domain app.id %}',
```

Comment:
Shouldn't this be different than `appSettingsURL`? Missing a `/releases`, maybe?